requirements.txt

```diff
@@ -2,8 +2,7 @@
 astral==2.2
 coloredlogs==15.0.1
 feedparser==6.0.5
-flask-restplus==0.13.0
-Flask==2.0.1
+flask-restx==0.5.0
```

Comment:
happy to see that here, since fask-restx seems to be the better choice 👍

Reply:
While testing, I received v3.4.1 as latest version when installing flask-restx (instead of 0.5.0). Can you confirm v0.5.0 or v3.4.1?

Reply:
I can confirm version v0.5.0.

Take a look at the github page (0.5.0 is the latest):
https://github.com/python-restx/flask-restx